docs/data-ai/capture-data/capture-sync.md

```diff
@@ -40,738 +40,93 @@ aliases:
-| `mongo_capture_config.database` | string | Optional | When `mongo_capture_config.uri` is non empty, changes the database data capture will write tabular data to. <br> Default: `"sensorData"`   |  <p class="center-text"><i class="fas fa-times" title="no"></i></p> |
-| `mongo_capture_config.collection` | string | Optional | When `mongo_capture_config.uri` is non empty, changes the collection data capture will write tabular data to.<br> Default: `"readings"`   |  <p class="center-text"><i class="fas fa-times" title="no"></i></p> |
-| `cache_size_kb` | float | Optional | `viam-micro-server` only. The maximum amount of storage bytes (in kilobytes) allocated to a data collector. <br> Default: `1` KB. |  <p class="center-text"><i class="fas fa-check" title="yes"></i></p> |
-| `file_last_modified_millis` | float | Optional | The amount of time to pass since arbitrary files were last modified until they are synced. Normal <file>.capture</file> files are synced as soon as they are able to be synced. <br> Default: `10000` milliseconds. |  <p class="center-text"><i class="fas fa-times" title="no"></i></p> |
-
-{{< /expand >}}
-
-## Configure data capture for individual resources
-
-You can capture data for any {{< glossary_tooltip term_id="resource" text="resource" >}} that supports it, including resources on {{< glossary_tooltip term_id="remote-part" text="remote parts" >}}.
+## Configure data capture and sync for individual resources
 
-Configure data capture for individual resources in their configuration by:
+1. Navigate to a configured {{< glossary_tooltip term_id="resource" text="resource" >}} in **Builder** mode in the [Viam app](https://app.viam.com)
+1. Find the **Data capture** section in the resource panel.
+1. Click **+ Add method**.
 
-- Selecting which resource methods to capture data from
-- Setting the capture frequency for each method
+   {{< imgproc src="/tutorials/data-management/add-method.png" alt="Disable data capture toggle" resize="700x" >}}
 
-{{< tabs >}}
-{{% tab name="Regular" %}}
+1. If you see a warning **Capture disabled on data management service**, click **Enable capture on data management service**.
+1. Select a **Method** to capture data from.
+1. Set the capture **frequency** in hertz, for example to `0.1` with `ReadImage` on a camera to capture an image every 10 seconds.
```

Comment:
```suggestion
1. Set the capture **frequency** in hertz, for example to `0.2` with `ReadImage` on a camera to capture an image every 5 seconds.
```
[opt] Might be a more informative example because it's not just powers of 10